packages/iTwinUI-react/.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Why this file was changed at all? Also quetes changed too.

Reply:
The contents of this file are now in `eslint-preset.js`